sdk/forms/src/main/assets/InAppFormsTemplate.html

```diff
@@ -7,7 +7,9 @@
 >
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0, viewport-fit=cover"/>
+    <meta name="referrer" content="same-origin" />
```

Comment:
[info here](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referrer-Policy)

You wouldn't believe the other stuff I tried before figuring out this was really just a one-liner...

Reply:
From what I observed in webkit browsers, iOS shouldn't need this tag, although I doubt it'd hurt?